src/Uno.UWP/UI/ViewManagement/ApplicationView.macOS.cs

```diff
@@ -64,6 +66,24 @@ public void ExitFullScreenMode()
 			}
 		}
 
+
+		public bool TryResizeView(Size value)
+		{
+			VerifyKeyWindowInitialized();
+
+			var window = NSApplication.SharedApplication.KeyWindow;
+			var frame = window.Frame;
+			frame.Size = new CGSize(value.Width, value.Height);
+			window.SetFrame(frame, true, true);
+			return true;
+		}
+
+		public void SetPreferredMinSize(Size minSize)
+		{
+			var window = NSApplication.SharedApplication.KeyWindow;
```

Comment:
Should `VerifyKeyWindowInitialized()` also be called here?

Reply:
Right!